helm_deploy/hmpps-manage-adjudications-api/templates/housekeeping-cronjob.yaml

```diff
@@ -3,7 +3,7 @@ kind: CronJob
 metadata:
   name: hmppps-manage-adjudications-api-delete-orphaned
 spec:
-  schedule: "*/10 * * * *"
+  schedule: "0 5 * * 1-5"
```

Comment:
at 05:00 on every day-of-week from Monday through Friday https://crontab.guru/#0_5_*_*_1-5